lib/state_utils.js

```diff
@@ -1,437 +1,435 @@
-    break;
+      break;
     case Constants.FanMode_Medium_Low: raw = 0x33;
-    break;
+      break;
     case Constants.FanMode_Medium: raw = 0x34;
-    break;
+      break;
     case Constants.FanMode_Medium_High: raw = 0x35;
-    break;
+      break;
     case Constants.FanMode_High: raw = 0x36;
-    break;
+      break;
   }
   return raw;
 }
 
-function setSwingModeFromRaw( device, raw ){
-  if ( raw != Constants.NONE_VAL ){
-    let status = "";
-    switch ( raw ){
+function setSwingModeFromRaw(device, raw) {
+  if (raw !== Constants.NONE_VAL) {
+    let status = '';
+    switch (raw) {
       case 0x31: status = Constants.SwingMode_Off;
-      break;
+        break;
       case 0x41: status = Constants.SwingMode_SwingVertical;
-      break;
+        break;
       case 0x42: status = Constants.SwingMode_SwingHorizontal;
-      break;
+        break;
       case 0x43: status = Constants.SwingMode_SwingVerticalAndHorizontal;
-      break;
+        break;
       case 0x50: status = Constants.SwingMode_Fixed_1;
-      break;
+        break;
       case 0x51: status = Constants.SwingMode_Fixed_2;
-      break;
+        break;
       case 0x52: status = Constants.SwingMode_Fixed_3;
-      break;
+        break;
       case 0x53: status = Constants.SwingMode_Fixed_4;
-      break;
+        break;
       case 0x54: status = Constants.SwingMode_Fixed_5;
-      break;
+        break;
       default: return;
     }
-    const allowedValues = Selections.getSwingModeResult( device, false )
-    if ( allowedValues.modes.find( element => element === status )){
-       const cap = device.getStoreValue( Constants.StoredCapabilityTargetSwingMode )
-       device.setCapabilityValue( cap , status).catch(this.error);
+    const allowedValues = Selections.getSwingModeResult(device, false);
+    if (allowedValues.modes.find(element => element === status)) {
+      const cap = device.getStoreValue(Constants.StoredCapabilityTargetSwingMode);
+      device.setCapabilityValue(cap, status).catch(this.error);
     }
   }
 }
 
-function setSwingModeToRaw( device){
-  const cap = device.getStoreValue( Constants.StoredCapabilityTargetSwingMode )
-  const target_swing_mode = device.getCapabilityValue( cap )
+function setSwingModeToRaw(device) {
+  const cap = device.getStoreValue(Constants.StoredCapabilityTargetSwingMode);
+  const targetSwingMode = device.getCapabilityValue(cap);
 
   let raw = Constants.NONE_VAL;
-  switch( target_swing_mode ){
+  switch (targetSwingMode) {
     case Constants.SwingMode_Off: raw = 0x31;
-    break;
+      break;
     case Constants.SwingMode_SwingVertical: raw = 0x41;
-    break;
+      break;
     case Constants.SwingMode_SwingHorizontal: raw = 0x42;
-    break;
+      break;
     case Constants.SwingMode_SwingVerticalAndHorizontal: raw = 0x43;
-    break;
+      break;
     case Constants.SwingMode_Fixed_1: raw = 0x50;
-    break;
+      break;
     case Constants.SwingMode_Fixed_1: raw = 0x51;
```

Comment:
This is the `duplicate case label` warning, which I think is just a typo?
```suggestion
    case Constants.SwingMode_Fixed_2: raw = 0x51;
```